src/entrypoints/custom-panel.ts

```diff
@@ -86,7 +88,49 @@ function initialize(
       (err) => {
         // eslint-disable-next-line
         console.error(err, panel);
-        alert(`Unable to load the panel source: ${err}.`);
+        let errorScreen;
```

Comment:
We could also use `window.parent.customPanel.navigate` to get out of the iframe

Reply:
neat!